executor/tasks/task_processor.py

```
@@ -303,7 +303,77 @@ async def _run_in_background():
+
+                # Attempt to send error notification to frontend
+                # Use a new event loop since we're in a callback context
+                try:
+                    # Create a fresh emitter in case the original one is corrupted
+                    fallback_emitter = _create_emitter(task_data)
+
+                    # Try to send error via the fallback emitter
+                    async def _send_fallback_error():
+                        try:
+                            await fallback_emitter.error(
+                                f"Task execution failed unexpectedly: {str(exc)}",
```

Ruff (0.15.5)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)